Various dependency updates

generic-pers0n opened the pull request:
This PR updates a few things in the Flatpak:

- We now use runtime 25.08.
- `shared-modules` was updated, and we also now use LAME from the runtime rather than `shared-modules`.
- PortMidi was updated to 2.0.7.

Note that wxWidgets still remains on the latest stable version, 3.2.8.1. Once 1.3.5 is released, I'm looking forward to finally updating wxWidgets to 3.3.1. That might actually help #1 because wxWidgets is now using `GtkFileChooserNative` for `wxFileDialog`, which means it'll also automatically use portals for file selection.

generic-pers0n committed:
Update runtime to 25.08

Signed-off-by: Avery King <gperson@disroot.org>

generic-pers0n committed:
Update PortMidi to 2.0.8

Signed-off-by: Avery King <gperson@disroot.org>

generic-pers0n commented:
Looks like the commit was wrong, so I fixed that.

I'm hoping this succeeds so we can have a more up-to-date environment. The big update, however, will be when we release 1.3.5 and upgrade to wxWidgets 3.3.x so we can potentially reduce our open permissions.

generic-pers0n committed:
Remove LAME from manifest

Apparently it's already in the Freedesktop SDK, so there's no need to
provide it here.

Signed-off-by: Avery King <gperson@disroot.org>

generic-pers0n commented:
It looks like the build was failing on LAME issues, and since it appears that LAME is already in the Freedesktop SDK, I'll experiment with using LAME from there instead of providing our own. (Or maybe I just broke stuff, I don't know :joy:)

generic-pers0n commented:
Great! Now the error is with libid3tag. I will be making a new release of the library hopefully soon!

AsciiWolf commented on org.tenacityaudio.Tenacity.yaml:
```diff
@@ -32,13 +32,13 @@ add-extensions:
     merge-dirs: ladspa;lv2;vst;vst3
     subdirectories: true
     no-autodownload: true
   org.freedesktop.Platform.ffmpeg-full:
```
The `ffmpeg-full` runtime extension is no longer needed since `25.08`. It was replaced by `codecs-extra` that is installed automatically.

More info: https://bbhtt.in/posts/closing-the-chapter-on-openh264/

yakushabb committed:
Drop ffmpeg extension

> This is supposed to ship in Freedesktop SDK 25.08. The major change for app developers is that there is no longer any need to have something like 

Source: https://bbhtt.in/posts/closing-the-chapter-on-openh264/
